### PR TITLE
Post: Deep Dive Khaos

### PR DESCRIPTION
**Pull request by aggre**
I added a new post

### CODE REVIEW

File: src/ja/posts/deep-dive-khaos.md [hunk 1]
```diff
@@ -0,0 +1,404 @@
+---
+title: Deep Dive Khaos
+author: Aggre
+date: 2021-03-20
+socialImage: '/images/posts/deep-dive-khaos/ogp.png'
+level: 中級
```
Comment (kazu80) [inline src/ja/posts/deep-dive-khaos.md:6]:
```suggestion
level: 中級以上
```

File: src/posts/deep-dive-khaos.md [hunk 2]
```diff
@@ -0,0 +1,402 @@
+---
+title: Deep Dive Khaos
+author: Aggre
+date: 2021-03-20
+socialImage: '/images/posts/deep-dive-khaos/ogp.png'
+level: INTERMEDIATE
```
Comment (kazu80) [inline src/posts/deep-dive-khaos.md:6]:
```suggestion
level: EXPERIENCED
```

**Review by kazu80 — CHANGES_REQUESTED** [review]

File: src/ja/posts/deep-dive-khaos.md (hunk 1 above)
Comment (aggre) [reply src/ja/posts/deep-dive-khaos.md]: I fixed this

File: src/posts/deep-dive-khaos.md (hunk 2 above)
Comment (aggre) [reply src/posts/deep-dive-khaos.md]: I fixed this

**Review by kazu80 — APPROVED** [review]

**Pull request merged**